eva-server/src/main/resources/application.properties

```diff
@@ -17,4 +17,6 @@ spring.jmx.enabled = false
 springfox.documentation.swagger.v2.path=/webservices/api
 
 #spring.profiles.active=oauth2-security
-#security.oauth2.resource.user-info-uri = ...
\ No newline at end of file
+#security.oauth2.resource.user-info-uri = ...
+
+eva.mongo.collections.files=@eva.mongo.collections.files@
```

Comment:
mmm, migrated to spring repositories? I don't think it could or should be removed, this is the place where the repositories will look for the collection names.